src/devices/bus/cbus/pc9801_86.cpp

```diff
@@ -200,36 +269,40 @@ void pc9801_86_device::device_reset()
 //**************************************************************************
 
 
-uint8_t pc9801_86_device::opna_r(offs_t offset)
+u8 pc9801_86_device::opna_r(offs_t offset)
 {
 	if((offset & 1) == 0)
 		return m_opna->read(offset >> 1);
 	else // odd
 	{
-		logerror("PC9801-86: Read to undefined port [%02x]\n",offset+0x188);
+		logerror("%s: Read to undefined port [%02x]\n", this->tag(), offset + m_io_base);
```

Comment:
You shouldn’t really be putting the tag in `logerror` since it adds that for you automatically by default.  Maybe put `machine().describe_context()` instead?  (Several other cases of that in this file as well.)